octobot_trading/exchanges/types/future_exchange.py

```diff
@@ -195,7 +195,10 @@ def parse_position_status(self, status):
         :param status: the position raw status
         :return: the uniformized position status
         """
-        raise NotImplementedError("parse_position_status is not implemented")
+        try:
+            return octobot_trading.enums.PositionStatus(status)
+        except KeyError:
```

Comment:
I think it is a ValueError when an enum is not found

Reply:
Right